samples/MauiEmbedding/MauiEmbedding/Package.appxmanifest

```diff
@@ -7,19 +7,18 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="5E560C41-9B2D-4E0D-85AD-F88AC0002C15"
-    Publisher="O=TestHarness"
+    Name="MauiEmbedding"
+    Publisher="O=NickRandolph"
```

Comment:
```suggestion
    Publisher="O=Uno Platform"
```

Reply:
Actually can you check with a new project... this seems to have a bunch of nodes that have been removed and should be provided by Resizetizer now.